crates/grafbase-sdk/changelog/0.13.0.md

```diff
@@ -0,0 +1,3 @@
+# Breaking changes
+
+- Renamed `FieldDirectiveSite.field_name()` to `FieldDirectiveSite.name()` and similarly for all `*DirectiveSite` structs.
```

Comment:
Should the `impl IntoQueryAuthorization` also be in there?

Reply:
It's only the template that is changed to avoid a Rust warning so not really necessary, but thanks!